components/tikv_util/src/metrics/threads_linux.rs

```diff
@@ -334,7 +446,9 @@ mod tests {
         tids.iter()
             .find(|t| {
                 pid::io_task(pid, **t)
-                    .map(|io| io.wchar == name.len())
+                    // .map(|io| io.wchar == name.as_bytes().len())
+                    // Use io.wchar can't pass the test now, io.wchar was twice the length of name
```

Comment:
Seems it can be passed on our machine, is it related to kernel version? (our kernel version is 3.10.0)

Reply:
Maybe you need [page_size](https://docs.rs/page_size/0.4.1/page_size/) ?

Reply:
The test failed on Ubuntu 18.04 64bit. 
Thanks, I change to use page_size now.